### ISSUE TITLE
Metrics of pcp-pmda-sockets are missing help texts.

### ISSUE THREAD

**Issue by kurik** [performancecopilot/pcp#1278]
Metrics of pcp-pmda-sockets are missing help texts.

The following metrics of pcp-pmda-sockets are missing online help texts.
This is output of ```pminfo -t``` filtered to metrics without a help text:
```
network.persocket.filter One-line Help: Error: One-line or help text is not available
network.persocket.netid One-line Help: Error: One-line or help text is not available
network.persocket.state One-line Help: Error: One-line or help text is not available
network.persocket.recvq One-line Help: Error: One-line or help text is not available
network.persocket.sendq One-line Help: Error: One-line or help text is not available
network.persocket.src One-line Help: Error: One-line or help text is not available
network.persocket.dst One-line Help: Error: One-line or help text is not available
network.persocket.inode One-line Help: Error: One-line or help text is not available
network.persocket.uid One-line Help: Error: One-line or help text is not available
network.persocket.sk One-line Help: Error: One-line or help text is not available
network.persocket.cgroup One-line Help: Error: One-line or help text is not available
network.persocket.v6only One-line Help: Error: One-line or help text is not available
network.persocket.ts One-line Help: Error: One-line or help text is not available
network.persocket.sack One-line Help: Error: One-line or help text is not available
network.persocket.cubic One-line Help: Error: One-line or help text is not available
network.persocket.ato One-line Help: Error: One-line or help text is not available
network.persocket.mss One-line Help: Error: One-line or help text is not available
network.persocket.pmtu One-line Help: Error: One-line or help text is not available
network.persocket.rcvmss One-line Help: Error: One-line or help text is not available
network.persocket.advmss One-line Help: Error: One-line or help text is not available
network.persocket.cwnd One-line Help: Error: One-line or help text is not available
network.persocket.ssthresh One-line Help: Error: One-line or help text is not available
network.persocket.bytes_sent One-line Help: Error: One-line or help text is not available
network.persocket.bytes_retrans One-line Help: Error: One-line or help text is not available
network.persocket.bytes_acked One-line Help: Error: One-line or help text is not available
network.persocket.bytes_received One-line Help: Error: One-line or help text is not available
network.persocket.segs_out One-line Help: Error: One-line or help text is not available
network.persocket.segs_in One-line Help: Error: One-line or help text is not available
network.persocket.data_segs_out One-line Help: Error: One-line or help text is not available
network.persocket.data_segs_in One-line Help: Error: One-line or help text is not available
network.persocket.send One-line Help: Error: One-line or help text is not available
network.persocket.lastsnd One-line Help: Error: One-line or help text is not available
network.persocket.lastrcv One-line Help: Error: One-line or help text is not available
network.persocket.lastack One-line Help: Error: One-line or help text is not available
network.persocket.pacing_rate One-line Help: Error: One-line or help text is not available
network.persocket.delivery_rate One-line Help: Error: One-line or help text is not available
network.persocket.delivered One-line Help: Error: One-line or help text is not available
network.persocket.app_limited One-line Help: Error: One-line or help text is not available
network.persocket.reord_seen One-line Help: Error: One-line or help text is not available
network.persocket.busy One-line Help: Error: One-line or help text is not available
network.persocket.dsack_dups One-line Help: Error: One-line or help text is not available
network.persocket.rcv_rtt One-line Help: Error: One-line or help text is not available
network.persocket.rcv_space One-line Help: Error: One-line or help text is not available
network.persocket.rcv_ssthresh One-line Help: Error: One-line or help text is not available
network.persocket.minrtt One-line Help: Error: One-line or help text is not available
network.persocket.notsent One-line Help: Error: One-line or help text is not available
network.persocket.timer.str One-line Help: Error: One-line or help text is not available
network.persocket.timer.name One-line Help: Error: One-line or help text is not available
network.persocket.timer.expire_str One-line Help: Error: One-line or help text is not available
network.persocket.timer.retrans One-line Help: Error: One-line or help text is not available
network.persocket.skmem.str One-line Help: Error: One-line or help text is not available
network.persocket.skmem.rmem_alloc One-line Help: Error: One-line or help text is not available
network.persocket.skmem.wmem_alloc One-line Help: Error: One-line or help text is not available
network.persocket.skmem.rcv_buf One-line Help: Error: One-line or help text is not available
network.persocket.skmem.snd_buf One-line Help: Error: One-line or help text is not available
network.persocket.skmem.fwd_alloc One-line Help: Error: One-line or help text is not available
network.persocket.skmem.wmem_queued One-line Help: Error: One-line or help text is not available
network.persocket.skmem.ropt_mem One-line Help: Error: One-line or help text is not available
network.persocket.skmem.back_log One-line Help: Error: One-line or help text is not available
network.persocket.skmem.sock_drop One-line Help: Error: One-line or help text is not available
network.persocket.wscale.str One-line Help: Error: One-line or help text is not available
network.persocket.wscale.snd One-line Help: Error: One-line or help text is not available
network.persocket.wscale.rcv One-line Help: Error: One-line or help text is not available
network.persocket.round_trip.str One-line Help: Error: One-line or help text is not available
network.persocket.round_trip.rtt One-line Help: Error: One-line or help text is not available
network.persocket.round_trip.rttvar One-line Help: Error: One-line or help text is not available
```

The following metrics of pcp-pmda-sockets are missing full help texts.
This is output of ```pminfo -T``` filtered to metrics without a help text:
```

network.persocket.filter
Full Help: Error: One-line or help text is not available

network.persocket.netid
Full Help: Error: One-line or help text is not available

network.persocket.state
Full Help: Error: One-line or help text is not available

network.persocket.recvq
Full Help: Error: One-line or help text is not available

network.persocket.sendq
Full Help: Error: One-line or help text is not available

network.persocket.src
Full Help: Error: One-line or help text is not available

network.persocket.dst
Full Help: Error: One-line or help text is not available

network.persocket.inode
Full Help: Error: One-line or help text is not available

network.persocket.uid
Full Help: Error: One-line or help text is not available

network.persocket.sk
Full Help: Error: One-line or help text is not available

network.persocket.cgroup
Full Help: Error: One-line or help text is not available

network.persocket.v6only
Full Help: Error: One-line or help text is not available

network.persocket.ts
Full Help: Error: One-line or help text is not available

network.persocket.sack
Full Help: Error: One-line or help text is not available

network.persocket.cubic
Full Help: Error: One-line or help text is not available

network.persocket.ato
Full Help: Error: One-line or help text is not available

network.persocket.mss
Full Help: Error: One-line or help text is not available

network.persocket.pmtu
Full Help: Error: One-line or help text is not available

network.persocket.rcvmss
Full Help: Error: One-line or help text is not available

network.persocket.advmss
Full Help: Error: One-line or help text is not available

network.persocket.cwnd
Full Help: Error: One-line or help text is not available

network.persocket.ssthresh
Full Help: Error: One-line or help text is not available

network.persocket.bytes_sent
Full Help: Error: One-line or help text is not available

network.persocket.bytes_retrans
Full Help: Error: One-line or help text is not available

network.persocket.bytes_acked
Full Help: Error: One-line or help text is not available

network.persocket.bytes_received
Full Help: Error: One-line or help text is not available

network.persocket.segs_out
Full Help: Error: One-line or help text is not available

network.persocket.segs_in
Full Help: Error: One-line or help text is not available

network.persocket.data_segs_out
Full Help: Error: One-line or help text is not available

network.persocket.data_segs_in
Full Help: Error: One-line or help text is not available

network.persocket.send
Full Help: Error: One-line or help text is not available

network.persocket.lastsnd
Full Help: Error: One-line or help text is not available

network.persocket.lastrcv
Full Help: Error: One-line or help text is not available

network.persocket.lastack
Full Help: Error: One-line or help text is not available

network.persocket.pacing_rate
Full Help: Error: One-line or help text is not available

network.persocket.delivery_rate
Full Help: Error: One-line or help text is not available

network.persocket.delivered
Full Help: Error: One-line or help text is not available

network.persocket.app_limited
Full Help: Error: One-line or help text is not available

network.persocket.reord_seen
Full Help: Error: One-line or help text is not available

network.persocket.busy
Full Help: Error: One-line or help text is not available

network.persocket.dsack_dups
Full Help: Error: One-line or help text is not available

network.persocket.rcv_rtt
Full Help: Error: One-line or help text is not available

network.persocket.rcv_space
Full Help: Error: One-line or help text is not available

network.persocket.rcv_ssthresh
Full Help: Error: One-line or help text is not available

network.persocket.minrtt
Full Help: Error: One-line or help text is not available

network.persocket.notsent
Full Help: Error: One-line or help text is not available

network.persocket.timer.str
Full Help: Error: One-line or help text is not available

network.persocket.timer.name
Full Help: Error: One-line or help text is not available

network.persocket.timer.expire_str
Full Help: Error: One-line or help text is not available

network.persocket.timer.retrans
Full Help: Error: One-line or help text is not available

network.persocket.skmem.str
Full Help: Error: One-line or help text is not available

network.persocket.skmem.rmem_alloc
Full Help: Error: One-line or help text is not available

network.persocket.skmem.wmem_alloc
Full Help: Error: One-line or help text is not available

network.persocket.skmem.rcv_buf
Full Help: Error: One-line or help text is not available

network.persocket.skmem.snd_buf
Full Help: Error: One-line or help text is not available

network.persocket.skmem.fwd_alloc
Full Help: Error: One-line or help text is not available

network.persocket.skmem.wmem_queued
Full Help: Error: One-line or help text is not available

network.persocket.skmem.ropt_mem
Full Help: Error: One-line or help text is not available

network.persocket.skmem.back_log
Full Help: Error: One-line or help text is not available

network.persocket.skmem.sock_drop
Full Help: Error: One-line or help text is not available

network.persocket.wscale.str
Full Help: Error: One-line or help text is not available

network.persocket.wscale.snd
Full Help: Error: One-line or help text is not available

network.persocket.wscale.rcv
Full Help: Error: One-line or help text is not available

network.persocket.round_trip.str
Full Help: Error: One-line or help text is not available

network.persocket.round_trip.rtt
Full Help: Error: One-line or help text is not available

network.persocket.round_trip.rttvar
Full Help: Error: One-line or help text is not available
```

**Comment by kmcdonell** [comment]
As of today, this list of metrics with missing help text seems to have been reduced to: network.persocket.delivered and
network.persocket.reord_seen.

**Comment by goodwinos** [comment]
Thanks Jan and Ken, I'll figure out what those two metrics are and add help text later today

**Comment by goodwinos** [comment]
Resolved with c0ff11472c93dc4cee47cd87bea20f90b59527c5
